src/geometry.cpp

```diff
@@ -79,4 +81,25 @@ namespace geometry {
 	  return v;
 
 	}
+
+	std::vector<Cell> raytraceLine_conversion(const Cell& start, const Cell& end) {
+
+	  // Get the interpolated pixels bresenham
+	  const auto pixelsAsInts = bresenham_conversion(start.x, start.y, end.x, end.y);
+
+	  // Convert std::vector<std::vector<int>> to std::vector<Cell>
```

Comment:
Is this all the original function did?

Reply:
Added a max length feature, although not smartly

Reply:
:disappointed: